test/server/server_corpus/clusterfuzz-testcase-minimized-server_fuzz_test-5714049408172032

```diff
@@ -0,0 +1,272 @@
+cluster_manager {
+  load_stats_config {
+    api_type: GRPC
+    grpc_services {
+      google_grpc {
+        target_uri: "18446744073709551617"
+        call_credentials {
```

Comment:
I think most of the `call_credentials` entries are unneeded. Can you prune the extraneous parts of this to minimize the size of the test case?

Reply:
Yup, I removed all of the `call_credentials` entries in the latest commit, so the test case is now only 18 lines (down from 272).